indico/modules/events/management/templates/_settings.html

```diff
@@ -21,6 +21,7 @@
         <div class="icon {{ icon }}"></div>
         <div class="text">
             {{ caller() }}
+            {{ template_hook('event-settings-section-extra', event=event, name=name) }}
```

Comment:
Looking at this again, maybe calling it `section` is better...

```suggestion
            {{ template_hook('event-settings-section-extra', event=event, section=name) }}
```